scripts/build_model.py

```diff
@@ -38,7 +38,7 @@ def aggregate_scores(
     if not row:
       continue
     feature = row.split('\t')[0]
-    feature_group, feature_content = feature.split(':')
```

Comment:
This could also be `split(':', 1)`, right? `partition` is fine with me, just wanted to check the intention.

Reply:
TBH, I just learned the `maxsplit` arg. I changed the code to keep `split`.